packages/arb-rpc-node/nitroexport/crossdb.go

```diff
@@ -124,6 +137,66 @@ func (c *CrossDB) importBlock(ctx context.Context, blockNumber uint64) error {
 	return err
 }
 
+func (c *CrossDB) OutboxBatchesExported() (uint64, error) {
+	return c.ethDB.Ancients()
```

Comment:
This function seems to be never called but I think it returns the number of blocks exported, not outbox batches. Looks like it can be removed in favor of `BatchesExported`